src/Controller/TiqrAppApiController.php

```diff
@@ -311,4 +332,20 @@ private function loginAction(Request $request, string $notificationType, string
 
         return new Response('AUTHENTICATION_FAILED', Response::HTTP_FORBIDDEN);
     }
+
+    private function registerSecureDevice(
+        string $notificationAddress,
+        TiqrUserInterface $user,
+        Response $responseObject
+    ): void {
+        // @TODO Valideren bij Pieter / Michiel of deze fallback er wel in moet
```

Comment:
Whether a notificationAdress is sent depends on the Tiqr client. A client may not send a notification address, for instance because it could not get one from the Apple/Google notification service. For an authentication, the notification address is send by the tiqr client as part of the OCRA response. It does not matter how the Tiqr authentication was started - using a QR code, or using a push notification - both the QR code and the push notification and contain the same Tiqr authentication URI.

It does not make sense to the the cookie when no notification address is present IMO, because that means that the client probably can't receive a push notification. We should update the notificationaddress after each successful authentication.

Reply:
Discussed: The tiqr app always sends an notificationAddress, so only use the one we receive.

TODO: Remove fallback to storage.

Reply:
:heavy_check_mark: This has been solved: Removed the fallback. 
If the Tiqr app does not send the notificationAddress, a cookie is not set.